Makefile

```diff
@@ -75,6 +75,9 @@ full-db-test: ensure-test-db
 clients-test:
 	cd test/clients; python clients_test.py
 
+types-test:
+	mypy --py2 .
```

Comment:
@mosen 
Should this be `py3` now?

Reply:
Sorry.. I'll fix this one and see if i can force py3 style annotations at the same time

Reply:
the version flag is omitted and the `python_version` option inside mypy.ini denotes 3.6 as the currently supported version to check types for.

Reply:
Thanks! I'm not sure that it makes any significant difference, but I believe we are aiming for Python 3.8. We can always adjust later if needed as I believe Py3.8 is backwards compatible with Py3.6 syntax.

I'll go ahead and resolve this.